component/azstorage/utils.go

```diff
@@ -138,6 +138,11 @@ func getAzDatalakeServiceClientOptions(conf *AzStorageConfig) (*serviceBfs.Clien
 
 // getLogOptions : to configure the SDK logging policy
 func getSDKLogOptions() policy.LogOptions {
+	// If BLOBFUSE_DISABLE_SDK_LOG env var is set to true, then disable the SDK logging
+	if os.Getenv("BLOBFUSE_DISABLE_SDK_LOG") == "true" {
+		return policy.LogOptions{}
+	}
+
```

Comment:
Add this in the `setSDKLogListener` method

Reply:
done